.circleci/config.yml

```diff
@@ -21,6 +21,10 @@ jobs:
           name: Run tests
           command: bazel test //...
       - run:
-          name: Test some package repositories
-          command: bazel build --jobs=2 \
-              @haskell_{aeson,language-c,lens,network}//...
+          name: Test some third-party packages
+          command: |
+              bazel build --jobs=2 \
+                  $(for p in `cat test-packages.txt`
```

Comment:
$(for p in $(<test-packages.txt)

Reply:
That didn't work in the CI environment which appears to have a stripped-down version of `/bin/sh`

Reply:
That's ridiculous. I'd fix the ci to have a known shell.

Reply:
Yeah, we're planning to revamp the docker image used to build this.  At the moment though everything's somewhat fragile so I'd rather change as little as possible to get CI passing again.